### ISSUE TITLE
[Bug] [Seatunnel-web] Wrong status code returned for incorrect login password

### ISSUE THREAD

**Issue by AshokRajShetty16** [apache/seatunnel#8385]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/seatunnel/issues?q=is%3Aissue+label%3A%22bug%22) and found no similar issues.

### What happened

When attempting to login to seatunnel web with an incorrect password, the system returns an incorrect status code. Instead of 401 unauthorized status code ,the system returns a different status code.

### SeaTunnel Version

2.3.8

### SeaTunnel Config

```conf
NA
```

### Running Command

```shell
Web login page with username and password
```

### Error Exception

```log
Status code 200, with error msg username and password not matched
```

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by arshadmohammad** [comment]
Yes, non-standard HTTP status codes are being used in Seatunnel-Web. Could you please explain how this impacts your use case? Along with the status code, there is also a message that clearly indicates the reason for failure.
